src/pybind/mgr/nfs/export.py

```diff
@@ -445,7 +445,7 @@ def create_export(self, fs_name, cluster_id, pseudo_path, read_only, path, squas
 
         if not self._fetch_export(pseudo_path):
             ex_id = self._gen_export_id()
-            user_id = f"{cluster_id}{ex_id}"
```

Comment:
Is this a breaking change?

Reply:
not AFAICT. new exports will get a nicer name; existing exports aren't touched.